README.md

```diff
@@ -174,6 +174,10 @@ The [`stm32-rs`](https://github.com/stm32-rs/stm32-rs) project has peripheral ac
 
 - [`ambiq-apollo3-pac`](https://crates.io/crates/ambiq-apollo3-pac) Peripheral access API for Ambiq Apollo3 microcontrollers (generated using svd2rust)
 
+### GigaDevice
+
+- [`gd32vf103-pac`](https://https://github.com/riscv-rust/gd32vf103-pac) Peripheral access API for GD32VF103 RISC-V microcontrollers (generated using svd2rust) - ![crates.io](https://img.shields.io/crates/v/gd32vf103-pac.svg)
```

Comment:
`https://https://`?

Reply:
Oops; resloved in https://github.com/rust-embedded/awesome-embedded-rust/pull/200/commits/a73a113f65de320e35cbd515041bc7be501f3187